### ISSUE TITLE
Exception when Hovering

### ISSUE THREAD

**Issue by edmondchui** [akempes/angular-rateit#2]
I have a very simple HTML page

``` html
  <ng-rate-it ng-model="rateIt" step="1" readonly="true"></ng-rate-it>
```

``` js
  $scope.rateIt = 1;
```

It renders without any problem. However, when I have the mouse pointer hovering the stars, I got the following exceptions in Javascript console:

``` js
"Error: event.originalTarget is undefined
$scope.onEnter@http://localhost:3000/consumer_ui/www/scripts/vendors/angular-rateit/dist/ng-rateit.js:111:3
$parseFunctionCall@http://localhost:3000/consumer_ui/www/scripts/vendors/angular/angular.js:12404:15
ngEventHandler/</callback@http://localhost:3000/consumer_ui/www/scripts/vendors/angular/angular.js:21566:17
$RootScopeProvider/this.$get</Scope.prototype.$eval@http://localhost:3000/consumer_ui/www/scripts/vendors/angular/angular.js:14466:16
$RootScopeProvider/this.$get</Scope.prototype.$apply@http://localhost:3000/consumer_ui/www/scripts/vendors/angular/angular.js:14565:18
ngEventHandler/<@http://localhost:3000/consumer_ui/www/scripts/vendors/angular/angular.js:21571:17
jQuery.event.special[orig].handle@http://localhost:3000/consumer_ui/www/scripts/vendors/jquery/dist/jquery.js:4765:11
jQuery.event.dispatch@http://localhost:3000/consumer_ui/www/scripts/vendors/jquery/dist/jquery.js:4434:15
jQuery.event.add/elemData.handle@http://localhost:3000/consumer_ui/www/scripts/vendors/jquery/dist/jquery.js:4121:6
"
```

**Comment by akempes** [comment]
@edmondchui I suspect that this is a cross browser issue... I will make a fix before the end of this week!

**Comment by akempes** [comment]
This issue is fixed since the 2.0.0 release. Thanks for reporting!

**Comment by edmondchui** [comment]
Thanks for the fix. I'll test it out.
